File: backend/src/main/java/io/blog/devlog/global/redis/message/CommentEmailMessage.java
package io.blog.devlog.global.redis.message;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor
@AllArgsConstructor
public class CommentEmailMessage {
    private String email;
    private String subject;
    private String postName;
    private String postUrl;
    private String commentContent;
    private String commentAuthor;
}

File: backend/mail-service/src/main/kotlin/io/blog/mail/redis/message/CommentEmailMessage.kt
package io.blog.mail.redis.message

import com.fasterxml.jackson.annotation.JsonProperty

data class CommentEmailMessage(
        @JsonProperty("email") val email: String,
        @JsonProperty("subject") val subject: String,
        @JsonProperty("postName") val postName: String,
        @JsonProperty("postUrl") val postUrl: String,
        @JsonProperty("commentContent") val commentContent: String,
        @JsonProperty("commentAuthor") val commentAuthor: String
)